importer/src/seattle.rs

```diff
@@ -157,21 +155,20 @@ pub fn match_parcels_to_buildings(map: &mut Map, shapes: &ExtraShapes, timer: &m
                 .and_then(|pts| Ring::new(pts).ok())
             {
                 let polygon = ring.into_polygon();
-                quadtree
-                    .insert_with_box(parcels_with_housing.len(), polygon.get_bounds().as_bbox());
+                quadtree.add_with_box(parcels_with_housing.len(), polygon.get_bounds());
                 parcels_with_housing.push((polygon, units));
             }
         }
     }
+    let quadtree = quadtree.build();
 
     let mut used_parcels: HashSet<usize> = HashSet::new();
     let mut units_per_bldg: Vec<(BuildingID, usize)> = Vec::new();
     timer.start_iter("match buildings to parcels", map.all_buildings().len());
     for b in map.all_buildings() {
         timer.next();
         // If multiple parcels contain a building's center, just pick one arbitrarily
-        for (idx, _, _) in quadtree.query(b.polygon.get_bounds().as_bbox()) {
-            let idx = *idx;
+        for idx in quadtree.query_bbox(b.polygon.get_bounds()) {
             if used_parcels.contains(&idx)
                 || !parcels_with_housing[idx].0.contains_pt(b.label_center)
```

Comment:
As the TODO above suggests, it'd be potentially nice to store the entire `Polygon` and have a convenience method for finding the thing that contains some point.